flloat/base.py

```diff
@@ -29,16 +31,35 @@ def negate(self) -> "Formula":
 
 
 class AtomicFormula(Formula, ABC):
-    """An abstract atomic formula."""
+    """An abstract atomic formula.
 
-    def __init__(self, s: Symbol):
-        """
-        Inintialize the atomic formula.
+    Both formulae and names can be used as atomic symbols.
+    A name must be a string made of letters, numbers, underscores, or it must
+    be a quoted string.
+    """
+
+    name_regex = re.compile(r'(\w+)|(".*")')
```

Comment:
Could you explain why this is different from the `name_regex` specified in `LTLfAtomic`? 
https://github.com/whitemech/flloat/pull/20/files#diff-b08f466ac095af653ce5dc9a48b028f9R88

I propose that we don't check what it's inside `QuotedFormula`, since the correctness of the value inside `QuotedFormula` is delegated to the `QuotedFormula` constructor. By doing so, we move `name_regex` in `LTLfAtomic` here, so we avoid redefinition.

Reply:
Any class that inherits from AtomicFormula must obey to this general naming convention (subclasses must call the super() constructors, though). The regex says: any string delimitated by " ", or a single word. LTLfAtomic, redefines the regex because it needs a more strict constraint: just lowercase words allowed.